.gitignore

```diff
@@ -1,2 +1,3 @@
+.idea
```

Comment:
I don't mind doing that here, but in general I think people are working with different IDEs and this is helping them as well..

But I really don't mind :)